specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -128,13 +128,13 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/locations/{location}/checkQuotaAvailability": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/locations/{location}/quotas/checkAvailability": {
```

Comment:
>/quotas/checkAvailability [](start = 95, length = 25)

this isn't a valid POST action URI.  Keep the URI the same, just update the operation ID to "Quotas_CheckQuotaAvailability" or something similar

Reply:
I change it back, but the problem now is:
```
WARNING (OperationIdNounVerb/R1001/SDKViolation): Per the Noun_Verb convention for Operation Ids, the noun 'Quotas' should not appear after the underscore. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.
    - file:///azure-rest-api-specs/specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json:137:8 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.VMwareVirtustream/locations/{location}/checkQuotaAvailability"].post.operationId)
```

Reply:
I see, it should just be `CheckQuotaAvailability`.

Reply:
The operationId is something that is SDK specific so I'm signing off from the ARM side.  @allenjzhang what is the intended operation ID here for a provider level action based on the various warnings?